pyproject.toml

```diff
@@ -38,15 +38,16 @@ pyyaml = ">=5.1"
 pytz = ">=2020"          # https://pythonhosted.org/pytz/#issues-limitations
 tzlocal = ">2.0, <3.0"   # https://github.com/regebro/tzlocal/blob/master/CHANGES.txt
 
+
 [tool.poetry.dev-dependencies]
 behave = "^1.2"
 mkdocs = "^1.0"
 black = {version = "^20.8b1",allow-prereleases = true}
 toml = ">=0.10"
 pyflakes = ">=2.2.0"
 pytest = ">=6.2"
+mock = "^4.0.3"
```

Comment:
Is there a reason for this? `mock` is in the standard library.

Reply:
removed and adjusted import declaration